packages/chrome/src/styled/sheet/StyledSheetFooterItem.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet_footer_item';
+
+export const StyledSheetFooterItem = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<ThemeProps<DefaultTheme>>`
+  margin: auto ${props => props.theme.space.base * 2.5}px;
```

Comment:
I just compared to the spec'd designs, and this margining is off – leaving the end item incorrectly positioned with respect to the padding. Please compare with https://github.com/zendeskgarden/react-components/blob/main/packages/modals/src/styled/StyledFooterItem.ts which applies the correct margins.

Reply:
Good call on this, updated!